language_formatters_pre_commit_hooks/pretty_format_java.py

```diff
@@ -23,7 +23,16 @@ def get_url(_version):
                 version=_version,
             )
 
-    return download_url(get_url(version))
+    url_to_download = get_url(version)
+    try:
+        return download_url(get_url(version), 'ktlint{version}.jar'.format(version=version))
```

Comment:
nit: this file has a mixture of single and double quotes. Is this intended?

Reply:
Not really intended, sorry for that.
Going to fix it for now even if I'm planning to move the codebase to use black (so we won't discuss about them anymore)